cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. logs should not show password during migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_associated_tags_before_and_after_migration_department_accounting_kind():
+    """
+    title : OSP: Test associated tags before and after migration
+    (Ex: Department:Accounting)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with a VM tagged to some category
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Migrated Vm should have the same tag associated.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vmware65_vm_with_multiple_nics():
+    """
+    title : OSP: vmware65-Test VM with multiple NICs with single IP (IPv6 to first
+    NIC and IPv4 to second)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with VM with multiple nics
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_retry_plan():
+    """
+    OSP: Test retry plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan so that it fails
+            3. Retry migration
+        expectedResults:
+            1.
+            2.
+            3. Migration starts
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_user_can_download_post_migration_ansible_playbook_log():
+    """
+    title: OSP: Test user can download post migration ansible playbook log
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan and run ansible playbooks
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Post migration logs should be downloadable
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_vm_owner_before_and_after_migration_remains_same():
```

Comment:
RHV usecase automated already, same can be used for OSP too, remove this since we are fixing OSP failures next week  ->  `test_single_vm_migration_power_state_tags_retirement`